libs/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-options/dataset-options.component.ts

```diff
@@ -13,13 +14,14 @@ import { DataLayerColorService } from '../../data-layer/data-layer-color.service
 })
 export class DatasetOptionsComponent implements OnInit {
   _dataset?: Dataset;
+  showAdvancedOptions: boolean = true;
```

Comment:
looks like this is unused now that you are using async pipe

Reply:
Okay, I will remove it and test it.